app/src/main/java/org/dhis2/usescases/programEventDetail/ProgramEventDetailPresenter.java

```diff
@@ -55,15 +55,15 @@ public void init() {
                 .observeOn(schedulerProvider.ui())
                 .subscribe(
                         catComboUid -> view.showCatOptComboDialog(catComboUid),
-                        Timber::e
+                        t8 -> Timber.e(t8)
                 )
         );
         compositeDisposable.add(eventRepository.featureType()
                 .subscribeOn(schedulerProvider.io())
                 .observeOn(schedulerProvider.ui())
                 .subscribe(
                         view::setFeatureType,
-                        Timber::e
+                        t7 -> Timber.e(t7)
                 )
```

Comment:
what is t?

Reply:
I weas debugging an error.  Chaning it back to Timber::e